src/SFA.DAS.ApplyService.Application/Apply/UpdatePageAnswers/UpdatePageAnswersHandler.cs

```diff
@@ -12,296 +12,296 @@
+            {
+              answer.Value = "Yes";
+            }
+          }
         }
 
-//        private static void MarkSequenceAsCompleteIfAllPagesComplete(ApplicationSequence sequence, List<Sequence> workflow)
-//        {
-//            sequence.Complete = sequence.Sections.SelectMany(s => s.Pages).All(p => p.Complete);
-//
-//            if (!sequence.Complete) return;
-//
-//            var nextSequences = sequence.NextSequences;
-//            foreach (var nextSequence in nextSequences)
-//            {
-//                if (nextSequence.Condition != null)
-//                {
-//                    var answers = sequence.Sections.SelectMany(s => s.Pages).SelectMany(p => p.PageOfAnswers[0].Answers).ToList();
-//                    if (answers.Any(a =>
-//                        a.QuestionId == nextSequence.Condition.QuestionId &&
-//                        a.Value == nextSequence.Condition.MustEqual))
-//                    {
-//                        workflow.Single(w => w.SequenceId == nextSequence.NextSequenceId).Active = true;
-//                    }
-//                }
-//                else
-//                {
-//                    workflow.Single(w => w.SequenceId == nextSequence.NextSequenceId).Active = true;
-//                }
-//            }
-//        }
+        pageAnswers.Answers.Add(answer);
+
+      }
+
+      return validationPassed;
     }
+
+    private static bool ExistingUpload(string questionId, List<PageOfAnswers> pagePageOfAnswers)
+    {
+      return pagePageOfAnswers.Any(poa => poa.Answers.Any(a => a.QuestionId == questionId && a.Value != ""));
+    }
+
+    //        private static void MarkSequenceAsCompleteIfAllPagesComplete(ApplicationSequence sequence, List<Sequence> workflow)
+    //        {
+    //            sequence.Complete = sequence.Sections.SelectMany(s => s.Pages).All(p => p.Complete);
+    //
+    //            if (!sequence.Complete) return;
+    //
+    //            var nextSequences = sequence.NextSequences;
+    //            foreach (var nextSequence in nextSequences)
+    //            {
+    //                if (nextSequence.Condition != null)
+    //                {
+    //                    var answers = sequence.Sections.SelectMany(s => s.Pages).SelectMany(p => p.PageOfAnswers[0].Answers).ToList();
+    //                    if (answers.Any(a =>
+    //                        a.QuestionId == nextSequence.Condition.QuestionId &&
+    //                        a.Value == nextSequence.Condition.MustEqual))
+    //                    {
+    //                        workflow.Single(w => w.SequenceId == nextSequence.NextSequenceId).Active = true;
+    //                    }
+    //                }
+    //                else
+    //                {
+    //                    workflow.Single(w => w.SequenceId == nextSequence.NextSequenceId).Active = true;
+    //                }
+    //            }
+    //        }
+  }
 }
```

Comment:
I take it there's a small change in here somewhere? 🤔

Reply:
@MarkFCain ?

Reply:
Greg did spot a name logic thing that was all about using the display type for logic which we resolved with a single line addition to reinject the displaytype on the viewmodel if there's a validation issue.  Not sure why it's gone change crazy.  It was line 43 btw.

Reply:
43, that's what I was after. Ta.